packages/action-tracker/src/index.ts

```diff
@@ -15,14 +15,29 @@ import {
 export const buildEmbraceMiddleware = () => {
   return (store: IMiddleware<IDispatch, any>) =>
     (next: IDispatch<IAnyAction>) =>
-    (action: IAnyAction) => {
+    async (action: IAnyAction) => {
       if (!next || typeof next !== "function") {
         console.warn(
           "[Embrace] The state managment middleware was not applied succesfully",
         );
         return 0;
       }
       const startTime = new Date().getTime();
+      const attributes = {
+        name: action.type.toString().toUpperCase(),
+        payload_size: zip(action.payload || 0).length,
+        "emb.type": "sys.rn_action",
+        outcome: "INCOMPLETE",
+      };
+      const spanId = await NativeModules.EmbraceManager.startSpan(
+        "emb-rn-action",
+        startTime,
+      );
```

Comment:
requires a second parameter for parentSpanId

```suggestion
      const spanId = await NativeModules.EmbraceManager.startSpan(
        "emb-rn-action",
        undefined,
        startTime,
      );
```